storage/src/adb/benches/fixed_init.rs

```diff
@@ -118,20 +175,39 @@ async fn gen_random_kv<
     db.close().await.unwrap();
 }
 
+const VARIANTS: [&str; 4] = [
```

Comment:
Consider using enum here (with `.name()`): https://github.com/commonwarexyz/monorepo/blob/9bd1f9bb554d585ee994016a8ac1d6f0304ad25f/storage/src/archive/benches/utils.rs#L32-L46

Reply:
ah sorry misunderstood your previous comment about this. done.